src/Plugins/RpcServer/ParameterConverter.cs

```diff
@@ -134,6 +142,79 @@ private static object ConvertBlockHashOrIndex(JToken token)
         throw new RpcException(RpcError.InvalidParams.WithData($"Invalid block hash or index Format: {token}"));
     }
 
+    private static object ConvertSigner(JToken token)
+    {
+        if (token is JObject jObject)
+        {
+            try
+            {
+                return Signer.FromJson(jObject);
+            }
+            catch (FormatException)
+            {
+                throw new RpcException(CreateInvalidParamError<Signer>(token));
+            }
+        }
+        throw new RpcException(CreateInvalidParamError<Signer>(token));
+    }
+
+    private static object ConvertContractParameter(JToken token)
+    {
+        if (token is JObject jObject)
+        {
+            try
+            {
+                return ContractParameter.FromJson(jObject);
+            }
+            catch (FormatException)
+            {
+                throw new RpcException(CreateInvalidParamError<ContractParameter>(token));
+            }
+        }
+        throw new RpcException(CreateInvalidParamError<ContractParameter>(token));
+    }
+
+    private static object ConvertSignerArray(JToken token)
+    {
+        if (token is JArray jArray)
+        {
+            try
+            {
+                return jArray.Select(t => Signer.FromJson(t as JObject)).ToArray();
+            }
+            catch (FormatException)
+            {
+                throw new RpcException(CreateInvalidParamError<Signer[]>(token));
+            }
+        }
+        throw new RpcException(CreateInvalidParamError<Signer[]>(token));
+    }
```

Comment:
It's not enough to be compatible with the old implementation. Let's use `SignersFromJson` instead because:
* it ensures that the constraint of the number of signers is not violated
* it validates signer format via serialization

Reply:
i dont think we should do multiple task in one pr, if you think  that should be applied, you can do one in another pr after this one. Prs that are simple, complete, small, and focus on only one purpose can be reviewed easier, and processed faster.

Reply:
Not resolved.

Reply:
Its not related to this pr, will not be resolved here. Whoever think it is necessary to do it should do it by himself.